README.md

```diff
@@ -194,7 +194,8 @@ Example   | Description |
 [App Service](azure-cs-appservice) | Build a web application hosted in App Service and provision Azure SQL Database and Azure Application Insights.
 [Bot Service](azure-cs-botservice) | Build an Azure Bot Service hosted in Azure App Service.
 [Cosmos App Component](azure-cs-cosmosapp-component) | Use a reusable component to create globally-distributed applications with Azure Cosmos DB.
-[Functions - Raw](azure-cs-functions-raw) | Deploy a function to Azure Functions created from raw deployment packages in C#.
+[Functions - .NET on Consumption](azure-cs-functions-consumption) | Deploy a .NET Azure Function running on a Windows Consumption Plan.
+[Functions - Python on Linux App Service](azure-cs-functions-linux-app) | Deploy a Python Azure Function running on a Linux App Service Premium Plan.
```

Comment:
I think when people are using this C# dotnet Pulumi SDK to create a function, this function will probably also be a C# function.

So besides deploying a Python FunctionApp, I think it's useful and logical to also deploy a C# functionapp using this example.

Reply:
Yeah, maybe .NET devs are very focused on .NET. For TypeScript, people asked about many runtimes, even though their Pulumi programs are in TypeScript. I also took Python because Python works _only_ on Linux plans.